geonode/harvesting/harvesters/base.py

```diff
@@ -81,13 +85,38 @@ def check_availability(self, timeout_seconds: typing.Optional[int] = 5) -> bool:
         """Check whether the remote service is online"""
 
     @abc.abstractmethod
-    def get_resource(
+    def get_resource_type_class(self, resource_type: str):
```

Comment:
Please add the expected return type to this method's signature

Reply:
Updated @ricardogsilva